src/Compilers/CSharp/Portable/Syntax/ArgumentSyntax.cs

```diff
@@ -0,0 +1,27 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.ComponentModel;
+
+namespace Microsoft.CodeAnalysis.CSharp.Syntax
+{
+    public sealed partial class ArgumentSyntax
+    {
+        /// <summary>
+        /// Pre C# 7.2 back-compat overload, which simply calls the replacement property <see cref="RefKindKeyword"/>.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public SyntaxToken RefOrOutKeyword
+        {
+            get => this.RefKindKeyword;
+        }
+
+        /// <summary>
+        /// Pre C# 7.2 back-compat overload, which simply calls the replacement method <see cref="Update"/>.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public ArgumentSyntax WithRefOrOutKeyword(SyntaxToken refOrOutKeyword)
+        {
+            return this.Update(this.NameColon, refOrOutKeyword, this.Expression);
```

Comment:
Perhaps `return WithRefKindKeyword(refOrOutKeyword);` instead. (The comment mentions calling the replacement method.)